<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Roushan" failures="0" tests="4" name="Default test" time="0.898" errors="0" timestamp="2025-02-23T22:46:55 IST">
  <testcase classname="api.test.UserTest" name="@BeforeClass setupData" time="0.804">
    <failure type="java.lang.ClassCastException" message="class org.apache.logging.log4j.core.Logger cannot be cast to class freemarker.log.Logger (org.apache.logging.log4j.core.Logger and freemarker.log.Logger are in unnamed module of loader &amp;apos;app&amp;apos;)">
      <![CDATA[java.lang.ClassCastException: class org.apache.logging.log4j.core.Logger cannot be cast to class freemarker.log.Logger (org.apache.logging.log4j.core.Logger and freemarker.log.Logger are in unnamed module of loader 'app')
at api.test.UserTest.setupData(UserTest.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setupData -->
  <testcase classname="api.test.UserTest" name="TestPostUser" time="0.0">
    <skipped/>
  </testcase> <!-- TestPostUser -->
  <testcase classname="api.test.UserTest" name="TestGetUserByName" time="0.0">
    <skipped/>
  </testcase> <!-- TestGetUserByName -->
  <testcase classname="api.test.UserTest" name="testUpdateUserByName" time="0.0">
    <skipped/>
  </testcase> <!-- testUpdateUserByName -->
  <testcase classname="api.test.UserTest" name="testDeleteUserByName" time="0.0">
    <skipped/>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- Default test -->
